### ISSUE TITLE
HTML5 play button doesn't work sometimes

### ISSUE THREAD

**Issue by AwesomeDonkey** [YePpHa/YouTubeCenter#1473]
Is it a known issue where sometimes when attempting to watch a video (HTML5 player) the play button doesn't appear to work? It's pretty random and it seems to be auto-buffering correctly but no matter what the play button won't work. Manually refreshing the page will 'fix' the issue. But I'm not sure if this is known or not. I have this sneaking suspicion it might be related to the prevent auto-play feature but since I can't reproduce this on-demand I can't test to verify that.

This sometimes happens in Chrome, Internet Explorer, Nightly, Safari, etc. Operating system doesn't matter nor does using Adguard's userscript injection method, Greasemonkey or Tampermonkey.

Here's my log: https://gist.github.com/anonymous/4ce1b6cdf51c5719147e

**Comment by Yonezpt** [comment]
This happens to me now and then, even without any addon enabled. I suspect that -at least in my case- it is caused by token expiration. I try to play the video when my token has expired and since the token is no longer valid, YouTube returns a 403 error (if I recall correctly) on the stream, which I can see in the console. Refreshing fixes the problem since it creates a new valid token.

Happens to me more often when I have a video that has ended and is inactive for a long time and then I try to replay it.

If this is the same problem as the one you are having then it is a YouTube only issue (not so much an issue as it is a safety feature).

**Comment by AwesomeDonkey** [comment]
Odd, I can load up fresh videos and this happens to me upon the page load. I'll have to open the console next time this happens.

**Comment by Yonezpt** [comment]
If it happens with fresh videos then odds are that our issues are not the same, but do pay attention to the console for any errors or warnings whenever it happens again.

**Comment by AwesomeDonkey** [comment]
This has been happening a lot lately.

**Comment by Yonezpt** [comment]
Did you check the console to look for any error output? I find it very strange that this happens across multiple browsers. Does it still happen if YTC is disable? Because if does then that's a YouTube problem.

**Comment by AwesomeDonkey** [comment]
No errors and it works fine when disabled.

**Comment by Yonezpt** [comment]
Haven't been having this issue for quite some time, but our YouTube usage may vary and that might make a difference. Have you tried resetting the YTC settings to their default? Also make sure that you are using the latest dev version 495, just in case you haven't updated it yet.

One last thing to try is to also disable any other addon that you might be using, disable everything except YTC and check wether the issue persists or not, there's always a possibility that another addon might be causing conflict with YTC.

**Comment by AwesomeDonkey** [comment]
I always use the latest dev builds. I'm thinking it might have something to do with the Prevent auto-play features being enabled, but I need to test that further.

**Comment by AwesomeDonkey** [comment]
Yep, I'm now convinced that prevent auto-play is the cause of this issue. I've had the feature disabled for nearly 2 days now and all videos worked fine. Within 5 minutes of re-enabling the prevent auto-play feature, it happened again. The video would appear to be buffering, but the play button wouldn't work.

**Comment by evliu** [comment]
+1 one this issue. I usually go to my subscriptions, then open up a bunch of videos I want to watch in new background tabs. When I do it like that, that's what causes my video to not be playable, although i can skip around the video and it fully loads. I have to refresh each tab when I get to them.
